cfme/scripting/quickstart/system.py

```diff
@@ -149,13 +149,16 @@ def retry_install_once(command, shell):
         run_cmd_or_exit(command, shell=shell)
 
 
-def install_system_packages():
+def install_system_packages(debuginfo_install=False):
+    installed = False
     if INSTALL_COMMAND and REQUIRED_PACKAGES:
         retry_install_once(INSTALL_COMMAND + REQUIRED_PACKAGES, shell=True)
-    if DEBUG_INSTALL_COMMAND and DEBUG_PACKAGES:
-        retry_install_once('{} {}'.format(DEBUG_INSTALL_COMMAND, ' '.join(DEBUG_PACKAGES)),
-                        shell=True)
-    else:
+        installed = True
+    if DEBUG_INSTALL_COMMAND and DEBUG_PACKAGES and debuginfo_install:
+        retry_install_once('{} {}'.format(DEBUG_INSTALL_COMMAND, ' '.join(DEBUG_PACKAGES))
+                           ,shell=True)
+        installed = True
+    if not installed:
```

Comment:
The way the logic works here, this was only displaying if DEBUG_INSTALL_COMMAND and DEBUG_PACKAGES weren't set, and making debuginfo_install optional meant it was displayed every time. Original logic was not great anyway, with it not triggering on `INSTALL_COMMAND and REQUIRED_PACKAGES`